docs/guides/run-jobs-batch.mdx

```diff
@@ -55,18 +55,9 @@ with Batch(backend=backend):
   sampler = Sampler()
 ```
 
-When you start a batch,  you must specify a QPU. This can be done by specifying its name or by passing a `backend` object.
+When you start a batch,  you must specify a QPU by passing a `backend` object.
 
-**Specify a QPU by name**
-
-``` python
-backend = service.least_busy(operational=True, simulator=False)
-service = QiskitRuntimeService()
-with Batch(backend=backend):
-    ...
-```
-
-**Pass a `backend` object**
+Example:
 
 ``` python
 backend = service.least_busy(operational=True, simulator=False)
```

Comment:
Do we really need this section? The 2 code snippets right above already show you need to pass in a `backend` object.

Reply:
good point